packages/ui/components/balance-section/index.tsx

```diff
@@ -15,7 +13,7 @@ import language from 'language'
 const { debtManagement } = language
 
 interface Props {
-  friend: Friend
+  friend: any
```

Comment:
Usage of the `any` type *seems* like an anti-pattern in TypeScript... or at least, a justified exception with an explanatory comment. Out of curiousity, why loosen this type restriction here?

Reply:
This was an attempt to resolve a Typescript compilation warning, and was not meant to be permanent.